### PR TITLE
feat(locale): extend Hebrew (he)

### PR DESCRIPTION
**Pull request by orinaveh**
## **Description**
As a Hebrew speaker who had to use this great package, I found some issues with the Hebrew locales. In this PR, I tried to fix as many of those issues as possible and make the data generated in Hebrew more realistic. 

#### **Buidling Number**
I Added common Israeli pattern for semidetached house's Building numbers.

 #### **Cities**
I Added a list of Israeli cities from our local MOI database.
Therefore I removed the prefix_city file.

 #### **Countries**
I updated the list as it seemed to be missing many countries.
In addition, I updated the default country to 'Israel' (instead of 'State of Israel') (lit.)

 #### **State**
I inserted the official regions of Israel to the list.

 #### **Streets**
In Hebrew, there are no suffixes to street's name, so I replaced the suffix with prefix file that includes common prefixes.
In addition, I took a list of streets names from some of cities in Israel and added that to `street_name.ts`

 #### **Direction Abbreviation**
I translated the values.

### **Cell Phones  & Phone numbers**

I added the Israeli patterns for Cell Phones and Phone numbers

### **Color**

The translation had [Niqqud](https://en.wikipedia.org/wiki/Niqqud) although using Niqqud is very uncommon in modern Hebrew, so I removed it.

#### **Genres**
I refined the translation of some of the genres

#### **First & Last Names**
Some names appeared twice or had a suffix of the word 'Family' between parentheses. I also added missing white-spaces.

#### **Prefix**
I translated the values.

### CODE REVIEW

File: src/locales/he/music/genre.ts [hunk 1]
```diff
@@ -3,8 +3,8 @@ export default [
   'רוק מטאלי',
   'פופ',
   'אלקטרוני',
-  'עממית',
-  'עולם',
+  'מוזיקת עם',
```
Comment (ST-DDT) [inline src/locales/he/music/genre.ts:6]: I'm unable to look this up using google translate: `music m`
Can you check whether you typed this one correctly and what the respective English translation is supposed to be?

**Review by ST-DDT — COMMENTED** [review]
I reviewed half of this PR and will do the rest tomorrow.
It looks good so far, I only had one word that I couldn't very using google translate.

**Comment by ST-DDT** [comment]
Thanks for your contribution!

File: src/locales/he/music/genre.ts (hunk 1 above)
Comment (orinaveh) [reply src/locales/he/music/genre.ts:6]: https://translate.google.co.il/?hl=iw&sl=iw&tl=en&text=%D7%9E%D7%95%D7%96%D7%99%D7%A7%D7%AA%20%D7%A2%D7%9D&op=translate

Google translate automatically detected `מוזיקת עם` as Yiddish term for some reason 😕

File: src/locales/he/phone_number/formats.ts [hunk 2]
```diff
@@ -1,22 +1,8 @@
...
-  '!##-!##-#### x#####',
-  '(!##) !##-#### x#####',
-  '1-!##-!##-#### x#####',
-  '!##.!##.#### x#####',
+  '02-#######',
+  '03-#######',
+  '04-#######',
+  '08-#######',
+  '09-#######',
+  '077-#######',
```
Comment (ST-DDT) [inline src/locales/he/phone_number/formats.ts:7]: Could you add a variant containing the international telephone prefix `+972` (?)?

**Review by ST-DDT — CHANGES_REQUESTED** [review]
Looks good to me.

File: src/locales/he/phone_number/formats.ts (hunk 2 above)
Comment (orinaveh) [reply src/locales/he/phone_number/formats.ts:7]: > Could you add a variant containing the international telephone prefix `+972` (?)?

@ST-DDT I have added the International format for both cell and landline numbers.

**Review by ST-DDT — APPROVED** [review]

**Review by import-brain — APPROVED** [review]

**Pull request merged**